pkg/apis/hive/v1alpha1/clusterimageset_types.go

```diff
@@ -0,0 +1,130 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// ClusterImageSetSpec defines the desired state of ClusterImageSet
+type ClusterImageSetSpec struct {
+	// hiveImage is the Hive image to use when installing or destroying a cluster.
```

Comment:
So in recent times, the case of new api type comments has been changed to reflect the names of the external json field names, instead of the Go names, given that those comments are used for external documentation (as in https://github.com/openshift/api/blob/master/operator/v1/types.go#L141-L155). However, that is not very evenly enforced. In our openshift API I see mostly the Go-style casing. I think that whatever we do we should be consistent about it, so I'm okay going to the Go-style case just to keep internal consistency.